src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.styles.js

```diff
@@ -1,5 +1,15 @@
-import styled, { css } from 'styled-components';
-import colors from '../../styles/colors';
+import styled, { css, keyframes } from 'styled-components';
+import { spacings, fonts, colors } from '../../styles/styles';
+
+const progressBar = keyframes`
+  0% {
+    transform: scaleX(0);
+  }
+
+  100% {
+    transform: scaleX(1);
+  }
+`;
```

Comment:
doesn´t work use only `to {
    transform: scaleX(1);
  }` ?

Reply:
It doesn't work like that, you have to indicate the start